### ISSUE TITLE
Presentation Feedback/Notes

### ISSUE THREAD

**Issue by IanTDuncan** [IanTDuncan/MealTime#21]

**Comment by bdshanks-28** [comment]
**IRB Peer Review**
A well designed box that makes points clear. Elevator statement seemed a little overcomplicated. Good detailed description.

**Aquatic Rig Peer Review:**
Elevator statement was a little too long and detailed for what it was supposed to be. Had a good business model canvas.

**Optilive Peer Review**
Elevator test statement has a great hook at the start but it was vague after that.
The box was very well designed though and clear to see all their main points.
Sounds like it will still have to be able to mostly have a backend for real estate people I'm not hearing much about. A lot of focus on the user finding a house but how would a seller use this app?
A lot of information seems heavily reliant on third party information.

**Comment by bdshanks-28** [comment]
**IRB Peer Review**
A well designed box that makes points clear. Elevator statement seemed a little overcomplicated. Good detailed description.

**Aquatic Rig Peer Review:**
Elevator statement was a little too long and detailed for what it was supposed to be. Had a good business model canvas.

**Optilive Peer Review**
Elevator test statement has a great hook at the start but it was vague after that.
The box was very well designed though and clear to see all their main points.
Sounds like it will still have to be able to mostly have a backend for real estate people I'm not hearing much about. A lot of focus on the user finding a house but how would a seller use this app?
A lot of information seems heavily reliant on third party information.

**Comment by aaleksandraristic** [comment]
**OptiLiv**
- Very nice box design and logo, it would definitely catch my eye if I am looking for something like that!
- Good presentation and main points. I like the idea of mock-ups, they would help a lot while you develop your idea.
- The Business Canvas Model is nice and simple, but very hard to follow, no text background color is needed - difficult to read it.
- Awesome technology diagram, important and useful!
- I am not sure about that database storage, but there is probably some easier way to make it work because the amount of data is enormous.

**Aquatic Rig**
- Very nice and interesting logo.
- Good website design for data visualization, keep it as simple as possible so it will be easy for each user to understand it.
- The Business Canvas Model is nicely formatted.
- There is a lot of text in your presentation, try to keep it short and visualize the main points (easier for an audience to follow)

**IRB**
- The best box design so far, and an outstanding logo!
- I like your idea for color coding for the Business Canvas Model.
- The main points of your software abilities are nicely done.

**Comment by Cade5480** [comment]
OptiLiv Review:
I really like the name and I thought your elevator test statement was very relatable and enticing to me personally. The design you guys showed was really unique and relaxing which is effective because in your case you want the user to be relaxed in their new rental space or when searching for one. I also like the overall design images of your app, the gps map is good, Google Maps would probably your best bet when implementing it into your app. Business Canvas Model was easy to look at and straight forward. The technology stack diagram was a little cluttered and for me didnt really get the technical idea across. Overall though i like idea and design so far.

BlueRover Review:
I like the name and logo off the bat, it is very simple, but clean. The vision statement was confusing, but that should be an easy fix for you guys. The box you guys also made was neat and well put together so good job.

IRB Review:
Statements and everything was super easy to follow and straight forward. The business canvas model looks very organized and well put together. The box design you guys has was really nice as well as the box design slides you had in your presentation. The detailed feature description was clear and concise. Overall i think it was a nice presentation. I think Morris and Dixon need to do a better job at communicating with you.

**Comment by BeepDroid** [comment]
**OptiLiv Peer Review**
Really gorgeous box design, clean and well done.
Adore the logo! Gives off the perfect feeling for the service you're providing.
I like the way you've done the stack diagram, but the communication between third party and first party is a huge part of it, and does seem a bit confusing as to how the application will work overall.
Elevator statement was pretty to the point.
Great use of the map!

**Aquatic Rig Peer Review**
Love the look of the logo. Combines the bot aspect and the wave aspect really well.
Elevator test statement is a little too long, but it does get the point across.
Clear and well communicated business model.
It was a bit hard to follow along completely.

**IRB Peer Review**
Amazing box composition! Love how it was put together and presented.
Color coded business model is a spectacular ide and communicated the different sections very effectively.
Presentation is very well put together visually and communication wise.
Clear breakdown of operation requirements and the hardware you'll need.
